src/rascal/math/utils.hh

```diff
@@ -33,6 +33,7 @@
 
 #include <cmath>
 #include <cstdint>
+#include <iostream>
```

Comment:
I would rather not have <iostream> in a public header. Can you replace the writing to `std::cout` by an exception where needed?

We could also add a small logging/warning framework for this.

Reply:
I want a warning not a termination of the program. The simplest way to do that is cout. What would you propose ?

Reply:
#245 

Let's keep the code as-is for now, and replace it later